Update activity_payment_actvitiy.xml to include better hint

Sarah160 opened the pull request.

Sarah160 committed:
Update activity_payment_actvitiy.xml

frankwang0098 commented:
This is fine, but please review and approve the other pr first. Then merge main into ur branch then i will approve

Sarah160 committed:
Merge branch 'main' into checkoutPayment

Sarah160 committed:
cartactvity updates from both promo code actvities

Sarah160 committed:
old promo code updates cartactivity

Sarah160 committed:
promo code activity updates cart activity

Sarah160 committed:
user has to change address and promocode before placing order

Sarah160 committed:
payment is updated to cart activity when added to paymentactvity

Sarah160 committed:
card should be added before order is placed

Sarah160 committed:
user does not need to enter promocode

I changed the code so the user does not have to enter a promo code to order but a card, an address and choose a delivery option
